contracts/payloads/IGP102/PayloadIGP102.sol

```diff
@@ -264,6 +266,54 @@ contract PayloadIGP102 is PayloadIGPMain {
         }
     }
 
+    // @notice Action 6: Update all the Maintenance Auths with Team Multisig
+    function action6() internal isActionSkippable(6) {
+        // update all the maintenance auths with Team Multisig
+        {
+            // Dex Fee Auths
+            address oldDexFeeAuth = 0xE3e18c563d11ced9B0c9cb8dD0284CF4442bC06a;
+            address newDexFeeAuth = 0x7BD48D505A195d2d3B90263b7E4DB78909b817D3;
+
+            // Remove old dex fee auth
+            DEX_FACTORY.setGlobalAuth(oldDexFeeAuth, false);
+
+            // Add new dex fee auth
+            DEX_FACTORY.setGlobalAuth(newDexFeeAuth, true);
+        }
+        {
+            // Rates Auths
+            address oldRatesAuth = 0x3eca30f7dB5AeAbD8757cE5Baf850dA8acA086Db;
+            address newRatesAuth = 0x1e6B029284dc2779F8FfBD83a3a5aA00EdCE6ba4;
+
+            // Remove old rates auth
+            LIQUIDITY.setGlobalAuth(oldRatesAuth, false);
```

Comment:
for Liquidity, the interface is updateAuths, see https://github.com/Instadapp/inst-governance/blob/c473ebef6ae608d70c1939875c2a20516e342db2/contracts/payloads/IGP35/PayloadIGP35.sol#L619-L628